src/System.Private.CoreLib/shared/System/DateTime.cs

```diff
@@ -97,9 +107,9 @@ public partial struct DateTime : IComparable, IFormattable, IComparable<DateTime
         private const int DatePartMonth = 2;
         private const int DatePartDay = 3;
 
-        private static readonly int[] s_daysToMonth365 = {
+        private static readonly int[] DaysToMonth365 = {
```

Comment:
Nit: I believe `s_daysToMonth365` is the right name according to the coding conventions.